client/internal/dns/server.go

```diff
@@ -421,7 +421,14 @@ func (s *DefaultServer) updateLocalResolver(update map[string]nbdns.SimpleRecord
 }
 
 func getNSHostPort(ns nbdns.NameServer) string {
-	return fmt.Sprintf("%s:%d", ns.IP.String(), ns.Port)
+	log.Debugf("getNSHostPort: %s : %d", ns.IP.String(), ns.Port)
+	if ns.IP.Is4() {
+		return fmt.Sprintf("%s:%d", ns.IP.String(), ns.Port)
+	}
+
+	n := fmt.Sprintf("[%s]:%d", ns.IP.String(), ns.Port)
+	log.Debugf("formated ns: %s", n)
```

Comment:
I think these two debug log is too much. We will print out "formatted ns:..." in case of ipv6 only.